lib/psql.js

```diff
@@ -11,11 +11,11 @@ function * exec (db, query) {
   debug(query)
   let env = Object.assign({}, process.env, {
```

Comment:
Oh I see. Well, if this is consistent with existing behavior, +1 on leaving it as is.